src/components/timeline/TrackContextMenu.js

```diff
@@ -516,13 +523,29 @@ class TimelineTrackContextMenu extends PureComponent<Props> {
     );
   }
 
+  renderShowAllTracks() {
+    const { rightClickedTrack } = this.props;
+    if (rightClickedTrack !== null) {
+      return null;
+    }
+    return (
+      <React.Fragment>
+        <MenuItem onClick={this._showAllTracks}>
+          `Show all tracks`
```

Comment:
I believe the quotes shouldn't be here?
![image](https://user-images.githubusercontent.com/454175/104951004-ffcb3600-59c1-11eb-835e-f38c0b8bd9e6.png)

Also I see a lint error in this area. Running `yarn lint-fix-js` should fix this error.